pytorch/inference/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -26,14 +26,14 @@ ARG PYTHON_VERSION
 ARG MAMBA_VERSION=4.12.0-0
 
 # PyTorch Binaries
-ARG PT_E3_INFERENCE_URL
```

Comment:
are we releasing the cu113 e3 images? if so, we should make sure to change the dlc_major_version label, since we have already released PT 1.11 with a diff cuda version. this is tricky though because of the fact that cu115 was released earlier.

If there is no specific need to release e3 cu113, then we can just plan to release sagemaker. let's change this major version to "2" for now, regardless

Reply:
updated